### ISSUE TITLE
Clean install Grails 3.1.6 and ckeditor tags not recognised

### ISSUE THREAD

**Issue by goedef** [stefanogualdi/grails-ckeditor#26]
I am a happy user of the ckeditor plugin. Until now I was working with Grails 2.x. Before migrating my current applications to Grails 3.x I am testing all my favourite plugins.

When I create a clean application and add the following:
repositories {
    maven { url "http://dl.bintray.com/stefanogualdi/plugins" }
}
dependencies {
    compile "org.grails.plugins:ckeditor:4.5.4.0"
}

The ckeditor plugin is not "recognised" by Grails. Meaning tha the <ckeditor:xxxxx/> tags do not get converted to the expected links etc.

**Comment by goedef** [comment]
I tried to build the plugin myself locally. Gradle complaint about a missing "websiteUrl" definition and could not publish the plugin locally tomaven. I added a url and now it works locally.

**Comment by stefanogualdi** [comment]
I tried but I cannot reproduce the problem with a new clean project using grails 3.1.6.

**Comment by puneetbehl** [comment]
@goedef  I think it would be great if you share the sample project replicating the problem.

**Comment by mkobel** [comment]
I have the sames problem as @stefanogualdi : I have a project where ckeditor is not recognized and I could not yet reproduced the behaviour with an clean example project.

I just recognized one difference using IntelliJ : The broken project shows the `org.grails.plugins:ckeditor:zip:4.5.4.0` in the list of external libraries, the working example just shows `org.grails.plugins:ckeditor:4.5.4.0` ( without `:zip:` ).

But on the other hand, `./gradlew dependencies` shows in both projects just `org.grails.plugins:ckeditor:4.5.4.0`

**Comment by a200612** [comment]
Its a really bad idea to use the same version scheme for the Grails 2 and the Grails 3 plugin. I just figured out that the old Grails 2 plugin version 4.5.4.0 gets downloaded instead of the Grails 3 plugin version 4.5.4.0.

**Comment by a200612** [comment]
For me it helped to delete the Grails 2 version 4.5.4.0 from the $HOME/.m2 directory then he uses the Grails 3 version 4.5.4.0 from the $HOME/.gradle directory.

**Comment by stefanogualdi** [comment]
You are right @a200612, really a bad idea! It was a stupid mistake! Do you have any proposal on how to change the version scheme?
